readme.md

```diff
@@ -17,7 +17,7 @@
 
 ## What is Ruby? (5 minutes / 0:05)
 
-Ruby is a **server-side** programming language.  
+We've been living in the client-side world of development thus far, running javascript in our browser. Fullstack developers work not just with a front-end, like we've done up until now, but with the back-end, aka a server. Ruby is a **server-side** programming language. Learning Ruby will get us in the door of the world of server-side programming.
```

Comment:
- javascript -> Javascript
- fullstack -> "full stack" or "full-stack"
- can remove "like we've done up until now" since you've already made that point in the previous sentence
- can remove "aka a server" since you talk about "server-side" in the following sentence